### DISCUSSION TITLE
BackTop 传入 transparent 背景色依然存在背景色

### DISCUSSION THREAD

**Discussion by OnlyWick** [DouyinFE/semi-design#2124] [General]
我想让 `BackTop` 的背景变成透明, 就传入了 `transparent` 样式, 但是这对内部的 `button` 不生效。
![image](https://github.com/DouyinFE/semi-design/assets/53461157/f7827765-6fca-4f5b-9907-937d09c35e74)

在这里加一个 `transparent` 判断, 换成 `borderless` 会不会好点?

https://github.com/DouyinFE/semi-design/blob/29a9d7e115640dd1e9d8559609d9e4ee44dcfd12/packages/semi-ui/backtop/index.tsx#L95

**Comment by OnlyWick** [comment]
有解决方案吗?

如何可以加个判断, 我可以 PR :-)

**Comment by pointhalo** [comment]
children 你可以传react Node，直接传入。

         <BackTop style={style}>
                    <Button theme='borderless'>semi</Button>
          </BackTop>

不倾向于再去判断style的属性决定button的某个属性，这个太隐蔽了。

**Reply by OnlyWick** [reply]
这样感觉有点麻烦, 我需要重新填充 BackTop 的东西
